examples/clusters.yaml

```diff
@@ -10,13 +10,17 @@ metadata:
   name: guestbook
 spec:
   generators:
-  - clusters: {}
+  - clusters:
+      selector:
+        matchLabels:
+          argocd.argoproj.io/secret-type: cluster
```

Comment:
This example just matches *every* cluster argo knows about as we're re-using the argo label.